### DISCUSSION TITLE
Pico 2 support

### DISCUSSION THREAD

**Discussion by gusarSlavko** [shanteacontrols/OpenDeck#311] [General]
Hi,
Any plans for a Pico 2 support in the future?
Cheers

**Comment by Stamenkovic21** [comment]
Zdravo Slavko ,
jesi li sa  Balkana ?
Pozz Zoran

Am So., 20. Okt. 2024 um 20:18 Uhr schrieb gusarSlavko <
***@***.***>:

**Reply by gusarSlavko** [reply]
Jesam

**Comment by Stamenkovic21** [comment]
Ako mozemo da kontaktiramo mozda mi mozete pomoci oko midi-a.
***@***.***

pozz
Zoran

Am Di., 22. Okt. 2024 um 16:38 Uhr schrieb gusarSlavko <
***@***.***>:

**Reply by gusarSlavko** [reply]
Ne vidim mail. Postavite ovdje new discussion, pa cu pomoc ako znam i mogu. Pozdrav

**Comment by paradajz** [comment]
> Hi, Any plans for a Pico 2 support in the future? Cheers

No.

**Comment by paradajz** [comment]
Pico 2 is supported now, it just isn't on the master branch yet. Will be officially available this week.
